packages/bun-types/bun.d.ts

```diff
@@ -2214,6 +2214,17 @@ declare module "bun" {
      */
     idleTimeout?: number;
 
+    /**
+     * Total minutes of activity before a client is automatically disconnected.
+     *
+     * `0` disables this feature.
+     * Must be a number between 0 and 240.
+     *
+     * @default 0
+     * @since Bun v1.1.26
+     */
+    maxLifetime?: number;
```

Comment:
Minutes is obscure, can we do seconds or millis?

Reply:
I believe this is the current uWebSockets behavior.

https://unetworking.github.io/uWebSockets.js/generated/interfaces/WebSocketBehavior.html#maxLifetime